manifests/nodejs.yml

```diff
@@ -554,7 +565,13 @@ tests/:
     test_logs.py:
       Test_Standardization: missing_feature
       Test_StandardizationBlockMode: missing_feature
-    test_metastruct.py: missing_feature
+    test_metastruct.py:
+      Test_SecurityEvents_Appsec_Metastruct_Disabled:
+        '*': v2.2.0
+        nextjs: *ref_4_17_0
```

Comment:
what's version 4.17.0 ? first appsec release for nextjs ?

Reply:
Yes, but I realize that this is wrong, right version is v2.2.0 for all the weblogs, because this test is using only the headers and it is global thing.